components/Search/Form.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react'
+import withT from '../../lib/withT'
+import Close from 'react-icons/lib/md/close'
+import { Field, mediaQueries } from '@project-r/styleguide'
+import { compose } from 'react-apollo'
+import withSearchRouter from './withSearchRouter'
+import { withAggregations } from './enhancers'
+import { DEFAULT_AGGREGATION_KEYS } from './constants'
+import InitState from './InitState'
+import { useDebounce } from '../../lib/hooks/useDebounce'
+import { css } from 'glamor'
+
+const styles = css({
+  paddingTop: 15,
+  [mediaQueries.mUp]: {
+    paddingTop: 40
+  }
+})
+
+const Form = compose(
+  withSearchRouter,
+  withAggregations,
+  withT
+)(
+  ({
+    urlQuery,
+    updateUrlQuery,
+    resetUrl,
+    dataAggregations,
+    t,
+    searchQuery,
+    setSearchQuery
+  }) => {
+    const [focusRef, setFocusRef] = useState(null)
+    const [formValue, setFormValue] = useState(urlQuery)
+    const [slowFormValue] = useDebounce(formValue, 200)
```

Comment:
This is for the backend not to get a crazy amount of requests through the withAggregations component, while keeping the frontend responsive